Documentation/Upgrade/rook-upgrade.md

```diff
@@ -48,21 +49,25 @@ those releases.
     official releases. Builds from the master branch can have functionality changed or removed at any
     time without compatibility support and without prior notice.
 
-## Breaking changes in v1.14
+## Breaking changes in v1.15
 
-* The minimum supported version of Kubernetes is v1.25.
-    Upgrade to Kubernetes v1.25 or higher before upgrading Rook.
-* The Rook operator config `CSI_ENABLE_READ_AFFINITY` was removed. v1.13 clusters that have modified
-    this value to be `"true"` must set the option as desired in each CephCluster as documented
-    [here](https://rook.github.io/docs/rook/v1.14/CRDs/Cluster/ceph-cluster-crd/#csi-driver-options)
-    before upgrading to v1.14.
-* Rook is beginning the process of deprecating CSI network "holder" pods.
+* Rook has deprecated CSI network "holder" pods.
```

Comment:
do we have to document upgrade with csi-operator experimental flag?

Reply:
Good question. My inclination is to have a separate doc for that this release since only a small percentage of users will use the CSI operator.